.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Please remove this here and add it to your global Git configuration, otherwise it will be included in the next pull request.

Reply:
removed